app/src/main/java/com/sdp/movemeet/chat/Message.java

```diff
@@ -0,0 +1,48 @@
+package com.sdp.movemeet.chat;
+
+import java.util.Date;
+
+public class Message {
+    private String messageUser;
+    private String messageText;
+    private String messageUserId;
+    private long messageTime;
```

Comment:
Why do you use a "long" type here?

Reply:
The time can't be change, it's set at current time when message is create, and so it cast in Date.
It's most easy to get a long, but the part for message in chat is not implemented yet, so it can change if we need.